src/pages/index.js

```diff
@@ -26,6 +28,13 @@ const profileModalDescription = profileEditModal.querySelector(
   "#profile-description-input"
 );
 
+//Avatar
+const avatarEditModal = document.querySelector("#profile-avatar-modal");
+const avatarEditForm = avatarEditModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms